lib/ash_cloak.ex

```diff
@@ -55,7 +57,8 @@ defmodule AshCloak do
         do_encrypt_and_set(changeset, key, value)
       end
     else
-      raise "Attempted to encrypt and set attribute #{inspect(key)} for resource #{inspect(changeset.resource)}, but it is not configured for encryption."
+      message = "Attempted to encrypt and set attribute #{inspect(key)} for resource #{inspect(changeset.resource)}, but it is not configured for encryption."
+      raise Ash.Error.Changes.InvalidAttribute, field: key, message: message
```

Comment:
We should create a specific error for this, i.e `AshClak.Errors.NoSuchEncryptedAttribute`. You can see the implementation of `InvalidAttribute` in Ash core for an example of how this is done 🙇

Reply:
@zachdaniel Code updated for your review.